### PR TITLE
tasks 1-9

### PR DESCRIPTION
**Pull request by yoncho2001**

### CODE REVIEW

File: app3/task1.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+            if (!double.TryParse(Console.ReadLine(), out number1)
+                || !double.TryParse(Console.ReadLine(), out number2))
+            {
+                Console.WriteLine("Incorrect input");
+                return;
+            }
+            if (number1 > number2)
+            {
+                double temp = number1;
```
Comment (GeorgiDyulgerov) [inline app3/task1.cs]: Can you come up with another solution without a temp variable?

File: app3/task2.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            {
+                Console.WriteLine("Incorrect input");
+                return;
+            }
+
+            if (score >= 1 && score <= 3)
+            {
+                outcome = (score * 10).ToString();
+            }
+
```
Comment (GeorgiDyulgerov) [inline app3/task2.cs]: Unneeded new line. below also

File: app3/task3.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
+namespace task3
+{
+    public class PlayCard
+    {
+        public static void Result()
+        {
+            string card = Console.ReadLine();
+            string outcome;
+
+            switch (card)
```
Comment (GeorgiDyulgerov) [inline app3/task3.cs:10]: Can you come up with another solution?  Write idea in comments.

File: app3/task5.cs [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+                || !double.TryParse(Console.ReadLine(), out number2)
+                || !double.TryParse(Console.ReadLine(), out number3)
+                || !double.TryParse(Console.ReadLine(), out number4)
+                || !double.TryParse(Console.ReadLine(), out number5))
+            {
+                Console.WriteLine("Incorrect input");
+                return;
+            }
+
+            double max = number1;
```
Comment (GeorgiDyulgerov) [inline app3/task5.cs:24]: Can you give me another soloution

File: app3/task7.cs [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+            {
+                between = number1;
+                min = number2;
+            }
+            else
+            {
+                between = number2;
+                min = number1;
+            }
+            if (between > number3)
```
Comment (GeorgiDyulgerov) [inline app3/task7.cs:34]: new line before another if block

File: app3/task9.cs [hunk 6]
```diff
@@ -0,0 +1,43 @@
+namespace task9
+{
+    public class IntDoubleString
+    {
+        public static void Result()
+        {
+            string input = Console.ReadLine();
+            string output = "";
+
+            switch (input)
```
Comment (GeorgiDyulgerov) [inline app3/task9.cs:10]: no default case?

**Review by GeorgiDyulgerov — CHANGES_REQUESTED** [review]

File: app3/task5.cs (hunk 4 above)
Comment (yoncho2001) [reply app3/task5.cs:24]: i can't think for another way

**Review by GeorgiDyulgerov — APPROVED** [review]

**Pull request merged**